sql/core/src/main/scala/org/apache/spark/sql/execution/command/cache.scala

```diff
@@ -17,16 +17,19 @@
 
 package org.apache.spark.sql.execution.command
 
+import java.util.Locale
+
 import org.apache.spark.sql.{Dataset, Row, SparkSession}
 import org.apache.spark.sql.catalyst.TableIdentifier
-import org.apache.spark.sql.catalyst.analysis.NoSuchTableException
```

Comment:
nit: I saw comments against change like this one in order to avoid issue in backporting the ticket.